packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -62,6 +64,10 @@ class ExpoImageView(context: ReactContext, private val requestManager: RequestMa
     // TODO: repeat mode handling
   }
 
+  internal fun setBlurRadius(blurRadius: Int) {
+    this.blurRadius = blurRadius
+  }
+
```

Comment:
Let's use kotlin properties https://kotlinlang.org/docs/properties.html
```suggestion
```

Reply:
I considered kotlin properties, but finally I followed the existing prop setters. I will change other prop setters in the near future.  👍